securedrop/source_app/main.py

```diff
@@ -221,8 +221,12 @@ def logout():
         if logged_in():
             msg = render_template('logout_flashed_message.html')
 
+            # save locale so we can restore it
+            user_locale = session['locale']
```

Comment:
The locale key does not always exist.

Reply:
oof, good call... `en_US` default it is

Reply:
it would be better to just not set it to avoid hardcoding the en_US default here

Reply:
if not set it should fallback to the DEFAULT_LOCALE set in config.py